lib/utils/transpile-es6.js

```diff
@@ -40,12 +40,36 @@ module.exports = function(tree, description, _options) {
     options.plugins = options.plugins.slice();
```

Comment:
Is this stil correct?  It looks like these should either be using `options.firstPass` or `options.secondPass`, or is there also still a `options.plugins`?